configx/provider_test.go

```diff
@@ -149,16 +149,15 @@ func TestAdvancedConfigs(t *testing.T) {
 			isValid: true,
 			envs: [][2]string{
 				{"DSN", "sqlite:///var/lib/sqlite/db.sqlite?_fk=true"},
-				{"TRACING_PROVIDER", "jaeger"},
-				{"TRACING_PROVIDERS_JAEGER_SAMPLING_SERVER_URL", "http://jaeger:5778/sampling"},
-				{"TRACING_PROVIDERS_JAEGER_LOCAL_AGENT_ADDRESS", "jaeger:6832"},
-				{"TRACING_PROVIDERS_JAEGER_SAMPLING_TYPE", "const"},
-				{"TRACING_PROVIDERS_JAEGER_SAMPLING_VALUE", "1"},
+				{"OTEL_EXPORTER_JAEGER_SAMPLING_SERVER_URL", "http://jaeger:5778/sampling"},
+				{"OTEL_EXPORTER_JAEGER_AGENT_HOST", "jaeger"},
+				{"OTEL_EXPORTER_JAEGER_AGENT_PORT", "6832"},
```

Comment:
Please see my other comment, imho we should have support for all of the 3 values:
- JAEGER_LOCAL_AGENT_ADDRESS
- JAEGER_AGENT_HOST
- JAEGER_AGENT_PORT